static/website/templates/get-question.html

```diff
@@ -265,17 +285,44 @@ <h5>Why is this answer being deleted?</h5>
             <div class="comments pull-right col-lg-11 col-md-11 col-sm-11 col-xs-11" >
                 <!-- displaying comments -->
                 {% for comment in answer.answercomment_set.all %}
+                {% if comment.is_active or MODERATOR_ACTIVATED %}
                 <div class="comment comment{{ comment.id }}" style="padding: 15px 15px 15px 5px;">
                     <div class="body" id="cbody{{ comment.id }}">
-                        <p>{{ comment.body| safe }}</p>
+                        {{ comment.body| safe }}
                     </div>
                     
                     <br>
-
-                    {% if user|is_moderator and MODERATOR_ACTIVATED %}
+                    {% if MODERATOR_ACTIVATED or user|can_edit:comment and answer|can_delete:comment.id %}
```

Comment:
The logic here is:

```
if MODERATOR_ACTIVATED:
   show the form
else if user|can_edit:comment and answer|can_delete:comment.id:
   show the form
```

so can_delete is checked only for the second part. is this logic correct?

Reply:
Yes, this is correct.